source/renderer/assets/codemirror/zettlr-plugin-render-math.js

```diff
@@ -56,33 +55,37 @@
       let newMarkers = []
 
       let line = cm.getLine(i)
-      if (!isMultiline && multilineMathRE.test(line)) {
+      let multilineMathMatch = multilineMathRE.exec(line)
+      let isMultilineStartOrEnd = multilineMathMatch !== null
+      if (!isMultiline && isMultilineStartOrEnd) {
         isMultiline = true
         fromLine = i
         eq = ''
-      } else if (isMultiline && !multilineMathRE.test(line)) {
+      } else if (isMultiline && !isMultilineStartOrEnd) {
         // Simply add the line to the equation and continue
         eq += line + '\n'
         continue
-      } else if (isMultiline && multilineMathRE.test(line)) {
+      } else if (isMultiline && isMultilineStartOrEnd) {
         // We have left the multiline equation and can render it now.
         isMultiline = false
         newMarkers.push({
           'curFrom': { 'ch': 0, 'line': fromLine },
-          'curTo': { 'ch': 2, 'line': i },
-          'eq': eq
+          'curTo': { 'ch': multilineMathMatch[1].length, 'line': i },
```

Comment:
That's the correction for the second point concerning the space in front of `$$`.